cves/kernel/CVE-2013-2140.yml

```diff
@@ -55,7 +55,26 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A vulnerability was found in the Linux kernel's driver for Xen Project virtual
+  disks in which a guest user could erase arbitrary blocks of memory called "sectors"
+  on read only disks, causing data loss.
+  
+  The Xen Project is an open source project focused on running multiple operating
+  systems on the same hardware at the same time. One part of how it accomplishes
+  this is allowing the creation of virtual disks that interface with the physical
+  storage disks so that each operating system does not have to. Virtual disks also
+  allow Xen to mediate each operating system's access to the actual disks.
+
+  The issue present in Linux's driver for these virtual disks was in the operation
+  for discarding or trimming sectors from the disk. Trimming in the context of disks
+  is when the operating system informs the disk that specific sectors on it are
+  no longer being used and can be erased. The problem with this operation in this
+  driver was that it did not check for the read only status of the disk before
+  performing the operation. If an admin provided a guest user read only access
+  to the disk, the user could still use the discard operation to delete arbitrary
+  data on the actual disk. This could impact not only the operating system using
+  the virtual disk, but other operating systems using Xen to access the actual disk.
```

Comment:
Very good and concise description of the issue, I like it.

Reply:
Thanks!